packages/backend/src/services/bid.ts

```diff
@@ -0,0 +1,42 @@
+import type { BidRepository } from '@repositories';
+import type { CreateBidDto } from '@types';
+import type { ProductRepository } from '@repositories';
+import { AuctionEndedError, ProductNotFoundError } from '@errors';
+import { toUtc } from '@helpers';
+import { LowBidPriceError } from '../error/product/low-bid-price-error';
+
+export class BidService {
+  private _bidRepository: BidRepository;
+
+  private _productRepository: ProductRepository;
+
+  constructor(
+    bidRepository: BidRepository,
+    productRepository: ProductRepository,
+  ) {
+    this._bidRepository = bidRepository;
+    this._productRepository = productRepository;
+  }
+
+  public async createBid(dto: CreateBidDto) {
+    const product = await this._productRepository.getById(dto.productId);
```

Comment:
auction. Add the check whether auction is active

Reply:
here it is
https://github.com/BinaryStudioAcademy/bsa-2022-vse-bude/blob/4678e3e80fa22f969a51e8bb6b7a4f555cbf7836/packages/backend/src/services/bid.ts#L28